pkg/config/validate.go

```diff
@@ -472,3 +473,19 @@ func (dc *DeploymentConfig) getStringValue(inputReference interface{}) (string,
 	}
 	return "", fmt.Errorf("the value %s is not a deployment variable or was not defined", inputReference)
 }
+
+func (dc *DeploymentConfig) testApisEnabled(validator validatorConfig) error {
```

Comment:
move function above `getStringValue` to be adjacent with other validators. 

If we expect this validator to be run first then it might even be best to put it first in the file to keep logical ordering.

Reply:
Done on `HEAD` of PR